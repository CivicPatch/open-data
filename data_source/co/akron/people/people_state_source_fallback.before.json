[
  {
    "name": "Brandon Hill",
    "roles": [
      "Mayor"
    ],
    "updated_at": "2025-07-09T00:12:48Z"
  },
  {
    "name": "Jared Jefferson",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-07-09T00:12:48Z"
  },
  {
    "name": "Crystann Benson",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-07-09T00:12:48Z"
  },
  {
    "name": "Braden Brent",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-07-09T00:12:48Z"
  },
  {
    "name": "Ariella Gonzales-Vondy",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-07-09T00:12:48Z"
  },
  {
    "name": "Jennifer Hansen",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-07-09T00:12:48Z"
  },
  {
    "name": "David Kembel",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-07-09T00:12:48Z"
  }
]