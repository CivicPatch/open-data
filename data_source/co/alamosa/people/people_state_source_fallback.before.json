[
  {
    "name": "Ty Coleman",
    "roles": [
      "Mayor"
    ],
    "updated_at": "2025-06-30T16:54:31Z"
  },
  {
    "name": "Jackie Vigil",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T16:54:31Z"
  },
  {
    "name": "Michael Carson",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T16:54:31Z"
  },
  {
    "name": "Jan Vigil",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T16:54:31Z"
  },
  {
    "name": "Liz Hensley",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T16:54:31Z"
  },
  {
    "name": "Dawn Krebs",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T16:54:31Z"
  },
  {
    "name": "Jamie Dominguez",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T16:54:31Z"
  }
]