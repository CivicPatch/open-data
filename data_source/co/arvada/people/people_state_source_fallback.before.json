[
  {
    "name": "Lauren Simpson",
    "roles": [
      "Mayor"
    ],
    "updated_at": "2025-06-27T20:08:56Z"
  },
  {
    "name": "Randy Moorman",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T20:08:56Z"
  },
  {
    "name": "Shawna Ambrose",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T20:08:56Z"
  },
  {
    "name": "John Marriott",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T20:08:56Z"
  },
  {
    "name": "Bob Fifer",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T20:08:56Z"
  },
  {
    "name": "Brad Rupert",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T20:08:56Z"
  },
  {
    "name": "Sharon Davis",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T20:08:56Z"
  }
]