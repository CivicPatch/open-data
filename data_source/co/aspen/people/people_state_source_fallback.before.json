[
  {
    "name": "Rachael E. Richards",
    "roles": [
      "Mayor"
    ],
    "updated_at": "2025-06-30T18:40:51Z"
  },
  {
    "name": "John Doyle",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T18:40:51Z"
  },
  {
    "name": "Bill Guth",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T18:40:51Z"
  },
  {
    "name": "Sam Rose",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T18:40:51Z"
  },
  {
    "name": "Christine Benedetti",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T18:40:51Z"
  }
]