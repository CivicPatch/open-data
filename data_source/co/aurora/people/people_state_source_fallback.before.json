[
  {
    "name": "Mike Coffman",
    "roles": [
      "Mayor"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Crystal Murillo",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Steve Sundberg",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Ruben Medina",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Stephanie Hancock",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Angela Lawson",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Fran√ßoise Bergan",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Alison Coombs",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Curtis Gardner",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Danielle Jurinsky",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  },
  {
    "name": "Amsalu Kassaw",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-27T19:32:56Z"
  }
]