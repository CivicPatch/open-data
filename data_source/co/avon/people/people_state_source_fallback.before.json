[
  {
    "name": "Tamra Underwood",
    "roles": [
      "Mayor"
    ],
    "updated_at": "2025-06-30T17:30:53Z"
  },
  {
    "name": "Rich Carroll",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T17:30:53Z"
  },
  {
    "name": "Chico Thuon",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T17:30:53Z"
  },
  {
    "name": "Ruth Stanley",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T17:30:53Z"
  },
  {
    "name": "Gary Brooks",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T17:30:53Z"
  },
  {
    "name": "Kevin Hyatt",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T17:30:53Z"
  },
  {
    "name": "Lindsay Hardy",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-30T17:30:53Z"
  }
]