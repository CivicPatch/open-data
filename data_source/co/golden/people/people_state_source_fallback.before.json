[
  {
    "name": "Laura Weinberg",
    "roles": [
      "Mayor"
    ],
    "updated_at": "2025-06-28T18:42:09Z"
  },
  {
    "name": "Lisa Vitry",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-28T18:42:09Z"
  },
  {
    "name": "Patty Evans",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-28T18:42:09Z"
  },
  {
    "name": "Robert Reed",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-28T18:42:09Z"
  },
  {
    "name": "Paul Haseman",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-28T18:42:09Z"
  },
  {
    "name": "Don Cameron",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-28T18:42:09Z"
  },
  {
    "name": "Bill Fisher",
    "roles": [
      "Council Member"
    ],
    "updated_at": "2025-06-28T18:42:09Z"
  }
]